Formula/pyside.rb

```diff
@@ -11,38 +11,39 @@ class Pyside < Formula
   end
 
   depends_on "cmake" => [:build, :test]
-  depends_on "llvm"
+  depends_on "llvm" => :build
```

Comment:
Why is it using `llvm` for, if it is only for build?

Reply:
`pyside` uses libclang to parse `qt` headers and generate the python bindings at compile time. libclang is not required by `pyside` at runtime. Initially, we had to include `llvm` because of opportunistic linkage of `libc++.1.dylib` from it, instead that from system. This creates, of course, a potential liability because this approach differs from upstream, which links to the library provided by the system. So, with @ilovezfs, we had an agreement to remove this runtime dependency as soon as possible (see the [5.11.0 PR](https://github.com/Homebrew/homebrew-core/pull/29699) or the [proposed then aborted PR](url
https://github.com/Homebrew/homebrew-core/pull/30229) for `libclang` for further info).